deploy-manage/deploy/kibana-reporting-configuration.md

```diff
@@ -1,31 +1,281 @@
 
-% - [ ] ./raw-migrated-files/kibana/kibana/secure-reporting.md
-% - [ ] ./raw-migrated-files/kibana/kibana/reporting-production-considerations.md
+* The functionality requires special OS dependencies which may not be available for all distributions and configurations of Linux.
+* It is subject to system resource configurations such as the limited number of file descriptors, allowed processes, and types of processes.
+* Linux versions that are in end-of-life phase are not supported.
+* Linux systems with `SELinux` or `fapolicyd` are not supported.
 
-% Internal links rely on the following IDs being on this page (e.g. as a heading ID, paragraph ID, etc):
+Before upgrading Kibana in a production environment, we encourage you to test your screenshotting use cases in a pre-production environment to make sure your hosts support our latest build of Chromium. For the most reliable configuration of PDF/PNG {{report-features}}, consider installing {{kib}} using [Docker](/deploy-manage/deploy/self-managed/install-kibana-with-docker.md), or using [Elastic Cloud](https://cloud.elastic.co).
+::::
 
-$$$reporting-chromium-sandbox$$$
+## Configuration overview
+
+To secure {{report-features}}, you must grant users access to reporting functionality and protect the reporting endpoints with TLS/SSL encryption. Additionally, you can install graphical packages on the operating system to enable screenshot capabilities in the {{kib}} server.
+
+Configuring reporting in your environment involves two main areas:
+
+### Granting users access to {{report-features}}
+
+Depending on your license, the type of users, and whether you prefer using the {{kib}} UI or API, there are multiple ways to grant access to reporting functionality. Choose the method that best fits your use case:
+
+* [Grant users access using {{kib}} UI](#grant-user-access)
+* [Grant access with the role API](#reporting-roles-user-api)
+* [Grant users access with a Basic license](#grant-user-access-basic)
+* [Grant access using an external provider](#grant-user-access-external-provider)
+
+### Applying system configuration
+
+The following configurations are required at {{es}}, {{kib}}, and OS levels to support {{report-features}}.
+
+::::{important}
+These settings apply only to **self-managed deployments**. Orchestrated deployments include this configuration by default. For more details on different deployment options, refer to [](/deploy-manage/deploy.md).
+::::
+
+* [Secure the reporting endpoints](#securing-reporting)
+* [Install the dependencies for the headless browser](#install-reporting-packages)
+* [Set the `server.host` for the headless browser](#set-reporting-server-host)
+* [Ensure {{es}} allows built-in templates](#reporting-elasticsearch-configuration)
+
+## Grant users access to reporting [grant-user-access]
+```yaml {applies_to}
+  deployment:
+    self: all
+    ece: all
+    eck: all
+    ess: all
+```
+
+### Grant users access using {{kib}} UI
```

Comment:
consider tabs for these processes

Reply:
done! feel free to review them before merging ;)